src/scripts/turnmarker.js

```diff
@@ -105,25 +90,28 @@ Hooks.on('updateCombat', async (combat, update) => {
 
 Hooks.on('deleteCombat', async () => {
     if (game.user.isGM) {
-        Marker.clearAllMarkers();
+        await Marker.clearAllMarkers();
     }
     MarkerAnimation.stopAnimation(animator);
 });
 
-Hooks.on('updateToken', (scene, updateToken, updateData) => {
+Hooks.on('updateToken', async (scene, updateToken, updateData) => {
```

Comment:
Adding async fixes an issue where last position of "active" player tile is desynced when reloading a game

Reply:
Turns out this async was not actually necessary.